rest/server_context.go

```diff
@@ -409,6 +430,12 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(config *DbConfig, useExisti
 
 	}
 
+	// Connect to the bucket and add the database:
+	spec, err := GetBucketSpec(config)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
Some of the functionality looks like it's now duplicated in GetBucketSpec and _getOrAddDatabaseFromConfig (pool/server/bucketname calculations - potentially just for use in logging?).

Reply:
Fixed by using the fields in GetBucketSpec